### DISCUSSION TITLE
При скачивании создаются папки, но без глав

### DISCUSSION THREAD

**Discussion by Meloni4** [lirix360/ReadmangaGrabber#62] [Q&A]
Скачивал манхву "Поднятие уровня в одиночку" и выбрал определённого переводчика, а после быстро скачались все главы и проверив, я понял, что создались папки без глав. Вот не понятно как это исправить. 

P.S: Проверял на другой манхве и там всё было нормально. 
Может проблема в том, что я выбираю определённого переводчика ?

**Comment by lirix360** [comment]
Скачивание одностраничной манхвы на данный момент не поддерживается.
